File: TrabalhoFinal/src/servidor/Esqueleto.java
package servidor;

public class Esqueleto {

    CalculadoraCiclista calculadora = new CalculadoraCiclista();

    public String IMC(String altura, String peso) {
        return calculadora.IMC(altura, peso);
    }

    public String alturaBike(String altura) {
        return calculadora.alturaBike(altura);
    }

    public String batimentosCardiacos(String batimentosTotal) {
        return calculadora.batimentosCardiacos(batimentosTotal);
    }

}

File: Trabalho_Final/TrabalhoFinal/src/servidorJS/Esqueleto.js
const CalculadoraCiclista = require('./Services');

class Esqueleto {
    calculadora = CalculadoraCiclista;

    IMC(argumentos) {
        const imc = new this.calculadora.IMC(...argumentos);
        return imc.calcIMC();
    }
    Bike(argumentos) {
        const bike = new this.calculadora.Bike(...argumentos);
        return bike.alturaBike();
    }
    Health(argumentos) {
        const health = new this.calculadora.Health(...argumentos);
        return health.batimentosCardiacos();
    }
}

module.exports = Esqueleto;